pkgs/top-level/all-packages.nix

```diff
@@ -18032,7 +18032,13 @@ with pkgs;
 
   bzrtp = callPackage ../development/libraries/bzrtp { };
 
-  c-ares = callPackage ../development/libraries/c-ares { };
+  c-ares = callPackage ../development/libraries/c-ares {
+    inherit (buildPackages) cmake;
```

Comment:
We should do this more often to fix cross builds.

Reply:
actually we should just not pass packages with callPackage

also i don't see a reason why cmake should be inherit here

Reply:
Oh, in the context of python-packages this would make sense but not all-packages.

Reply:
#196038

Reply:
yea, cmake should be getting added correctly as it's listed as a `nativeBuildInputs`. But i dont' see much harm in it either